src/main.py

```diff
@@ -181,12 +187,17 @@ async def on_message(message: discord.Message):
             await message.channel.send(lvl_up_message)
 
     for log_link in re.findall(r"https://smapi.io/log/[a-zA-Z0-9]{32}", message.content):
-        log_dict = requests.get(f"http://api.pil.ninja/smapi_log/endpoint?{log_link}").json()
+        log_dict = requests.get(f"{SERVER_URL}/log.php?{log_link}").json()
```

Comment:
Tried to run this, and the program throws an error `json.decoder.JSONDecodeError: Expecting value: line 1 column 1`. When I try and open that URL, there is no json file to parse, so something is up with the PHP code.

Reply:
Does the php show an error if you enable errors?